crates/bin/pcli/src/command/migrate.rs

```diff
@@ -30,6 +51,23 @@ pub enum MigrateCmd {
     /// minus any gas prices for the migration transaction.
     #[clap(name = "balance")]
     Balance,
+    /// Migrate balances from specified subaccounts to a destination wallet.
+    ///
+    /// All assets from the specified source subaccounts will be sent to the destination wallet.
+    /// A FullViewingKey must be provided for the destination wallet.
+    /// Gas fees will be paid from the source subaccount with the most fee token.
+    #[clap(name = "subaccount-balance")]
+    SubaccountBalance {
+        /// Range of source subaccount indices to migrate from (e.g., 0..17)
+        #[clap(long, required = true, value_parser = parse_range)]
+        from: std::ops::Range<u32>,
+        /// Destination FullViewingKey to migrate to
+        #[clap(long, required = true)]
+        to: String,
+        /// Only print the transaction plan without executing it (for threshold signing)
+        #[clap(long)]
+        plan_only: bool,
```

Comment:
The `--plan-only` option is very cool :ok_hand: I'd like to see more of this, preferably with derived Display impls so we can reuse liberally.